docker/run_container.sh

```diff
@@ -37,9 +37,9 @@ echo "Configuring APEL Server"
 ALLOWED_FOR_GET=
```

Comment:
Yes, I have a word document with what these variables do and what they are set to on kubernetes.

 I'll add a redacted copy to the admin.md aswell